scripts/custom_lint_checks.py

```diff
@@ -269,6 +269,14 @@ def visit_functiondef(self, node):
         self.check_functiondef_yields(node, node_doc)
 
     def check_functiondef_params(self, node, node_doc):
+        """Called for checking function definition parameter.
```

Comment:
This function does not check the parameters in the function definition instead it checks whether all parameters in the function definition are documented.
Similarly for others. Please change it accordingly.
Also, use something like: `Checks whether all parameters in a function definition are documented.`

Reply:
Only in the check_ functions, right?
do other functions starting with visit, etc have correct docstring. I mean do they tell what is being done correctly or are they also checking the documentation?

Reply:
The `check_` functions check the documentation whereas the `visit_` function actually visit each function node. Please read through the code. There are other `visit_` functions already documented which might help you to understand them. Thanks!

Reply:
Okkay. Done that. Thanks.